### DISCUSSION TITLE
Migrate to TailwindCSS

### DISCUSSION THREAD

**Discussion by 6543** [woodpecker-ci/woodpecker#2147] [Polls]
https://windicss.org/posts/sunsetting.html

we have to move to a maintained CSS framework and I would propose to use [TailwindCSS](https://github.com/tailwindlabs/tailwindcss) as WindiCSS originated from it

---
**:+1:, :-1: or comment**

**Comment by 6543** [comment]
Ok the dessision is clear ...

How can we convert that back into a tiket?

**Comment by qwerty287** [comment]
@6543 see https://github.com/woodpecker-ci/woodpecker/issues/2194
